vercel/turbo
# [DO NOT MERGE] `scan` command doc.

### Description

Documenting the new `scan` command

### Testing Instructions

Visit `/repo/docs/reference/command-line-reference/scan` to verify correctness.


Closes TURBO-2527
